spec/classes/foreman_proxy_content__pub_dir_spec.rb

```diff
@@ -10,11 +10,6 @@
       let(:pre_condition) do
         <<-PUPPET
         include foreman_proxy
-        class { 'foreman_proxy::plugin::pulp':
-          enabled          => false,
-          pulpnode_enabled => false,
-          pulpcore_enabled => false,
-        }
         PUPPET
       end
 
```

Comment:
This entire pre-condition is now redundant. `foreman_proxy_content` already has it but it was here because `foreman_proxy::plugin::pulp` needed it. Since that's gone, this can also go.
```suggestion
```

This is repeated in multiple tests so I'm not commenting it everywhere.